resources/js/components/people/ContactSearch.vue

```diff
@@ -49,7 +49,19 @@ export default {
       if (contact.item.id > 0) {
         window.location = contact.item.route;
       } else {
-        window.location = 'people/add';
+        const names = contact.item.keyword.split(' ').map(name =>
+          name.charAt(0).toUpperCase() + name.substr(1).toLowerCase()
```

Comment:
You might use `_.capitalize`. See https://lodash.com/docs/4.17.15#capitalize

Reply:
In the last commit this is resolved